src/modules/PackageSlideShow.rb

```diff
@@ -125,60 +121,90 @@ def InitPkgData(force)
       return if @init_pkg_data_complete && !force
 
       ResetPackageSummary()
-      # Reinititalize some globals (in case this is a second run)
-      @total_size_installed = 0
-      @updated_packages = 0
-      @installed_packages = 0
 
       total_sizes_per_cd_per_src = Pkg.PkgMediaSizes
       total_pkg_count_per_cd_per_src = Pkg.PkgMediaCount
 
       @total_size_to_install = ListSum(total_sizes_per_cd_per_src)
-      log.info "total_size_to_install: #{@total_size_to_install}"
       @total_pkgs_to_install = ListSum(total_pkg_count_per_cd_per_src)
+      @expected_total_download_size = Packages.CountSizeToBeDownloaded
       @init_pkg_data_complete = true
-    end
 
-    # Recalculate remaining times per CD based on package sizes remaining
-    # and data rate so far. Recalculation is only done each 'recalc_interval'
-    # seconds unless 'force_recalc' is set to 'true'.
-    #
-    # @param [Boolean] _force_recalc force recalculation even if timeout not reached yet
-    # @return true if recalculated, false if not
-    #
-    # @see SlideShow.next_recalc_time
-    # @see Yast2::SystemTime.uptime
-    def RecalcRemainingTimes(_force_recalc)
-      true
+      log.info "Total size to install: #{String.FormatSize(@total_size_to_install)}"
+      log.info "Expected download size: #{String.FormatSize(@expected_total_download_size)}"
+      log.info "Parallel download (initial): #{parallel_download?}"
+      nil
     end
 
-    # Switch unit to seconds if necessary and recalc everything accordingly.
-    # @return true if just switched from sizes to seconds, false otherwise
-    #
-    def SwitchToSecondsIfNecessary
-      false
-    end
+    # Check if package are downloaded in parallel to being installed.
```

Comment:
Typos: if packages are downloaded in parallel to be installed

Reply:
No, it's really meant like this: It's the download and the installation (of other packages) that are in parallel.